contracts/SSVNetworkViews.sol

```diff
@@ -70,17 +70,17 @@ contract SSVNetworkViews is
 
     function getOperatorById(
         uint64 operatorId
-    ) external view override returns (address, uint256, uint32, bool) {
+    ) external view override returns (address, uint256, uint32, bool, bool) {
         (
             address operatorOwner,
             uint64 fee,
             uint32 validatorCount,
             Snapshot memory snapshot
         ) = _ssvNetwork.operators(operatorId);
-        bool active;
-        if (snapshot.block != 0) active = true;
+        bool isPrivate = _ssvNetwork.operatorsWhitelist(operatorId) == address(0) ? false : true;
```

Comment:
not sure this is enough.  What if an operator is private (has a fee of 0) then removes his address, we will say that he is not private even though he is